package.json

```diff
@@ -44,6 +44,7 @@
     "@ngrx/store": "^5.2.0",
     "@ngrx/store-devtools": "^5.2.0",
     "@ngx-translate/i18n-polyfill": "^0.1.2",
+    "@types/plotly.js": "^1.35.0",
```

Comment:
Types should go to `devDependencies`. We do not want to bundle them with the application.